pkg/pillar/types/certinfotypes.go

```diff
@@ -5,16 +5,46 @@ package types
 
 import (
 	"encoding/hex"
+	"github.com/google/go-tpm/tpmutil"
+)
+
+// hashing algorithm used for a payload
+type ZHashAlgorithm uint8
```

Comment:
No 'Z' for types local to EVE. That should only be used in the API.

Reply:
And the existing CertHashTypeSha256First16 should work just fine so no need to introduce a differently named type.